pkg/psmdb/backup/pbm.go

```diff
@@ -469,3 +473,13 @@ func (b *pbmC) GetPITRChunkContains(ctx context.Context, unixTS int64) (*pbm.Opl
 
 	return c, nil
 }
+
+// Node returns replset node chosen to run the backup
+func (p *pbmC) Node() (string, error) {
+	lock, err := p.GetLockData(&pbm.LockHeader{Replset: p.rsName})
+	if err != nil {
+		return "", err
+	}
+
+	return strings.Split(lock.Node, ".")[0], nil
```

Comment:
should we double check if `lock.Node` is not an empty string? otherwise operator will panic

Reply:
Maybe you're right, I relied on it will be populated since we got no error. This info is read from the collection PBM creates. But I agree, we should be more protective here, it will not hurt.

Reply:
Can it happen that we collect this, but the node changes due to election?
Does maybe PBM store actual node somewhere so we can just collect it?
What happens in sharding where we can have multiple replicas?

Reply:
1. I guess it can, but that also applies to the PDM itself. We just collect the lock PBM places for its purposes.
2. It is stored in `admin.pbmLock`, this API call just reads that basically
3. We don't change or affect that, here we just read what PDB has written basically, nothing more.